ubuntu-advantage

```diff
@@ -58,13 +65,14 @@ Currently available are:
 
 Commands:
  version                   show the tool version
- status                    show current status of Ubuntu Advantage offerings
- enable-esm <token>        enable the ESM repository
+ status <OFFERING>         show current status of Ubuntu Advantage offerings
```

Comment:
This should be `[OFFERING]`, since it's not mandatory. I'm not very keen on the name `OFFERINGS`, even though it was used in `status` before. Perhaps `name`?

Reply:
yeah, name sounds better, fixed